<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(13, 13, 13);</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 700 15pt &quot;Futura&quot;;
color: rgb(173, 194, 255);
background-color: rgba(101, 52, 255, 104);
border-radius:20px
}
QPushButton:hover{
background-color: rgba(101, 52, 255, 204);
}
QPushButton:pressed{
	background-color: rgba(22, 22, 61, 189);
	color: rgb(219, 215, 255);
}</string>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signinButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>220</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 700 15pt &quot;Futura&quot;;
color: rgb(173, 194, 255);
	background-color: rgb(51, 68, 95);
border-radius:10px
}
QPushButton:hover{
background-color: rgba(101, 52, 255, 204);
}
QPushButton:pressed{
background-color: rgba(22, 22, 61, 189);
	color: rgb(219, 215, 255);
}</string>
    </property>
    <property name="text">
     <string>Sign In</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>do not have an account?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	
border: 1px solid grey;
border-radius:20px;
	background-color: rgba(17, 17, 17, 245);
padding: 10px
}
QLineEdit:hover{
border:1px solid white;
}
QLineEdit:pressed{
border:1px solid rgba(95, 139, 255, 219);
}

</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	
border: 1px solid grey;
border-radius:20px;
	background-color: rgba(17, 17, 17, 245);
padding:10px;
}
QLineEdit:hover{
border:1px solid white;
}
QLineEdit:pressed{
border:1px solid rgba(95, 139, 255, 219);
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="tryagain">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
